### PR TITLE
Accessing jump boxes via Concourse

### PR DESCRIPTION
**Pull request by rogeruiz**
Addresses #278 

Pairing with @LinuxBozo @jbarnicle

### CODE REVIEW

File: content/ops/troubleshooting-bosh.md [hunk 1]
```diff
@@ -5,49 +5,72 @@ menu:
...
+## Accessing jumpboxes
 
-If you're going to be accessing Bosh, you will need to intercept a jumpbox via Concourse.
+If you're going to be accessing Bosh, you will need to intercept a jumpbox via
+Concourse.
 
 ### Pre-flight checklist:
+
 1. Need to have access to Concourse for cloud.gov (e.g., https://ci.cloud.gov)
 1. Need to have `fly` installed locally
```
Comment (NoahKunin) [inline content/ops/troubleshooting-bosh.md]: Hyperlink on how to install fly?

Comment (mogul) [inline content/ops/troubleshooting-bosh.md]: It's in the bottom corner of ci.[fr.].cloud.gov, and yeah, we should point that out explicitly.

**Comment by mogul** [comment]
Noah asked that we include a high-level diagram to summarize what's happening here.

> I would not over optimize or work the diagram at this point. Just take a pic of a piece of paper/cocktail napkin/whiteboard.

I think we are after a sequence diagram or something that looks like the diagram Veris made for the RAR.

**Comment by rogeruiz** [comment]
The `<code>` blocks in this update are very `w i d e` and will need 18F/cg-style#98 to land in order to allow for better readability via horizontal scrolling.

File: content/ops/troubleshooting-bosh.md (hunk 1 above)
Comment (LinuxBozo) [inline content/ops/troubleshooting-bosh.md]: Pretty sure this is covered well enough in the next two lines?

Comment (rogeruiz) [inline content/ops/troubleshooting-bosh.md]: I'm going to be pointing to [the official documentation](https://github.com/concourse/fly#installing-from-the-concourse-ui-for-project-development) along with updating the description on our own for added clarity.

**Comment by rogeruiz** [comment]
@jbarnicle This should be ready for your walkthrough.

**Comment by rogeruiz** [comment]
@brittag Could you take a look at this documentation and see if it could be any clearer? Also, ☝️ the comment about adding a diagram. I have a diagram, but I'm not sure where it would be most helpful to add it.

**Comment by jbarnicle** [comment]
@rogeruiz - your stuff looks good.  Added a couple comments - if those look good, feel free to merge.

**Comment by LinuxBozo** [comment]
@rogeruiz @jbarnicle  Is this good to go now?

**Comment by jbarnicle** [comment]
@LinuxBozo - ok on my end.

**Comment by rogeruiz** [comment]
I will look into this later today.

**Comment by afeld** [comment]
Worked for me—thanks so much for writing this up!

**Pull request merged**